components/dashboard/src/projects/NewProject.tsx

```diff
@@ -368,9 +368,9 @@ export default function NewProject() {
                 <p className="text-gray-500 text-center text-base mt-12">
                     {loaded && noReposAvailable ? "Select account on " : "Select a Git repository on "}
                     <b>{selectedProviderHost}</b> (
-                    <a className="gp-link cursor-pointer" onClick={() => setShowGitProviders(true)}>
+                    <button className="gp-link cursor-pointer" onClick={() => setShowGitProviders(true)}>
                         change
```

Comment:
**issue(non-blocking)**: This will slightly affect how the link style is rendered but that's minor for now.

| BEFORE | AFTER |
|-|-|
| <img width="454" alt="link-before" src="https://user-images.githubusercontent.com/120486/167153574-b24c6573-27a1-479b-add8-97ad8a131582.png"> | <img width="454" alt="link-after" src="https://user-images.githubusercontent.com/120486/167153577-acb6dabf-a26b-4ae7-ad76-264c9f80995e.png"> |

Reply:
Good catch 🏅 !

I guess it stems from [`text-sm` in the Tailwind styles applied to the `button` element](https://github.com/gitpod-io/gitpod/blob/7b0acb66f47d47ee5533a8a6c3cba545a677272d/components/dashboard/src/index.css#L50) while `a` seems to just inherit as usual.

The upcoming button component should take it into account. If you need a volunteer for working on the component, you know where to find me 😊